### DISCUSSION TITLE
"Lua", not "LUA"

### DISCUSSION THREAD

**Discussion by Pharap** [tsunamayo/Starship-EVO#4300] [General]
As a long-time user of Lua, I feel it's my duty to point out that Lua is not an acronym and should not be written in all-caps.

Here's an explanation of the name taken from [the official Lua website](https://www.lua.org/about.html):

> ## What's in a name?
>
> "Lua" (pronounced **LOO-ah**) means "Moon" in Portuguese. As such, it is neither an acronym nor an abbreviation, but a noun. More specifically, "Lua" is a name, the name of the Earth's moon and the name of the language. Like most names, it should be written in lower case with an initial capital, that is, "Lua". Please do not write it as "LUA", which is both ugly and confusing, because then it becomes an acronym with [different meanings](http://acronyms.thefreedictionary.com/lua) for different people. So, please, write "Lua" right!

Hence I ask of you all: _if you see anyone getting it wrong, please correct them so we can stamp this misconception out early, before it creeps into the player base._

Thank you in advance to anyone helping to further this goal. Let's stop the problem before it starts.

As ever, I am incredibly grateful to see the work going into this game, and I very much look forward to the addition of Lua in the future.

---

(P.S. If anyone who is not familiar with Lua is looking to learn, I'm happy to provide learning resources and advice, or answer questions about the language, as well as point out a few other games that use Lua scripting for their modding facilities.)

**Comment by Briaireous** [comment]
Thought earths moon was named Luna

**Reply by Pharap** [reply]
I'm not sure if that's supposed to be a joke or not.

Assuming not, yes, 'Luna' is one name for Earth's moon, but a number of languages use slightly different variants of 'luna' as their word for moon (e.g. in French it's 'la lune'), and in Portuguese the word for moon is 'lua'. ([Lua was created at a Brazilian university](https://en.wikipedia.org/wiki/Lua_(programming_language)#History).)

---

It's just ocurred to me how fitting it is for a game about space to have a scripting language named after a kind of celestial body.

**Comment by Jorpis-Jorah** [comment]
I have a bad feeling people are going to be mad at this

**Reply by Pharap** [reply]
More likely they just won't care, and my pleas will be ignored.
If they were angry then I'd probably have been flooded with thumbs-down reactions by now.

It's probably a lost cause, but I'd rather try and fail than not try at all.

**Comment by Uncle-Ulty** [comment]
Just an observation. The language "Lua" is called that way, in Portuguese, because it was developed in Brazil, a Portuguese-speaker country...
